### PR TITLE
[RM Ch10]: Replacing NFVI and VIM

### LINKED ISSUE #1168: [RM] generic requirements of CNF in RM3.3 and RM4.1
@tomkivlin we have some observation in RM stream . It looks the requirements are too much VNF focused and not scoped adequately for CNF .

E.g 4.1 Capabilities and Performance Measurements we can find VIM performance but not  K8S platform capabilities part 

3.3 NFVI Management Software is another example we expect something like container management as well 
Similarly for K8S at minimum performance requirements , CRD's and CRI-O etc should be included

I am sure it is included somewhere can you ignore my negligence if so and help us understand how you are tacking it .

### PR DESCRIPTION
**Pull request by EditKoselak**
Partially closes #1168

### CODE REVIEW

File: doc/ref_model/chapters/chapter10.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ As the first component, the reference model must also be structured and iterated
 
 Going forward, this chapter shall be dedicated to identifying the challenges and gaps found in the course the development--to ensure the reference model is adding the strategic and tactical value intended over time. Should a challenge or gap not be identified and it is not already addressed in the model itself, the community may assume it will remain an unknown and, therefore, the community is welcomed to engage with and raise an issue with the appropriate working group(s) to close the gap. In this manner, the Reference Model can continuously improve.
 
-For Reference - Common NFVI Lifecycle Framework Diagram (Figure 10-1)
+For Reference - Common Cloud Infrastructure Lifecycle Framework Diagram (Figure 10-1)
 <p align="center"><img src="../figures/ch10-fig-10-1-common-nfvi_lifecycle_framework.png" alt="Framework" title="Framework" width="65%"/></p>
```
Comment (deebhatia) [inline doc/ref_model/chapters/chapter10.md:29]: Please fix the title in the image. It still uses the NFVI term.

Comment (EditKoselak) [reply doc/ref_model/chapters/chapter10.md:29]: The source file is missing from the artefacts folder. @jbeltranatl, @mf4716, @pgoyal01, could you please send it to us?

**Comment by rabi-abdel** [comment]
@kedmison can you please review/approve?

**Comment by rabiabdel** [comment]
i will approve under the assumption that the diagram will be updaged once the source image is found.

**Pull request merged**